src/auth/Cargo.toml

```diff
@@ -54,3 +54,4 @@ test-case   = "3"
 tokio       = { version = "1", features = ["macros", "rt-multi-thread", "test-util"] }
 tokio-test  = "0.4"
 url         = "2"
+mutants     = "0.0.3"
```

Comment:
I think we should start moving all such dependencies to the top-level `Cargo.toml` file and just use `mutants.workspace = true` here.  I am not asking you change the existing deps, just the new one.